cli/tests/integrations/test_ssl.py

```diff
@@ -0,0 +1,91 @@
+import os
```

Comment:
@tamarrow Can we test that overriding the CA works? I don't remember if we decided to punt on this.